config/test.exs

```diff
@@ -12,3 +12,14 @@ config :beacon, Beacon.Repo,
   ownership_timeout: 1_000_000_000
 
 config :beacon, ecto_repos: [Beacon.Repo]
+
+config :ex_aws,
+  access_key_id: "AKIAIOSFODNN7EXAMPLE",
+  secret_access_key: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
+  region: "us-east-1"
```

Comment:
Can we document that these are fake values? I read the code before the PR description and had a moment 😅. I could see future readers of the code doing the same and not having the PR description as context.

Reply:
> Can we document that these are fake values? I read the code before the PR description and had a moment 😅. I could see future readers of the code doing the same and not having the PR description as context.

Good point 😂

Reply:
Absolutely!

Reply:
done!